src/NUnitFramework/framework/Constraints/EmptyDirectoryConstraint.cs

```diff
@@ -31,9 +32,6 @@ namespace NUnit.Framework.Constraints
     /// </summary>
     public class EmptyDirectoryConstraint : Constraint
     {
-        private int files = 0;
-        private int subdirs = 0;
```

Comment:
I think these were originally up here at class-level in order to support the ability to, at one time, customizing the error message: https://github.com/nunit/nunit/pull/3623/files#diff-d32b7706fe57f7797f420ad3a90872e8R71

I've left that block untouched for now, but I'm happy to remove it if it's not needed. Based on the git history, I don't think it's ever been used.

Reply:
I think we should remove it, but please wait until the next reviewer (in case he/she disagrees)

Reply:
Agreed, we can remove these.